src/components/admin/ChannelConfiguration/channelSchema.ts

```diff
@@ -0,0 +1,13 @@
+import * as Yup from "yup";
+
+import { createNameSchema } from "forms/factory/createNameSchema";
+
+interface ChannelSchemaShape {
+  name: string;
+  iframeUrl: string;
+}
+
+export const channelSchema = Yup.object().shape<ChannelSchemaShape>({
+  name: createNameSchema({ name: "Name", withMin: true }),
+  iframeUrl: Yup.string().required(`URL is required`),
+});
```

Comment:
This file should be in `forms/` unless we have a new convention and I don't know about it, in that case ignore my comment.